src/containers/cooperation-details/cooperation-from-scratch/CooperationFromScratch.tsx

```diff
@@ -26,29 +30,42 @@ const CooperationFromScratch = () => {
   )
 
   return (
-    <Box sx={styles.root}>
-      <Box sx={styles.publishBlock}>
-        <Box>
-          <Box sx={styles.lockBlock}>
-            <LockOpenTwoToneIcon sx={styles.icon} />
-            <Typography sx={styles.lockTitle}>
-              {t('cooperationDetailsPage.publish')}
+    <Box>
+      <Box sx={styles.root}>
+        <Box sx={styles.publishBlock}>
+          <Box>
+            <Box sx={styles.lockBlock}>
+              <LockOpenTwoToneIcon sx={styles.icon} />
+              <Typography sx={styles.lockTitle}>
```

Comment:
no need to create own style property for icon and typographies
can write nested styles in parent lockBlock and use it.
you can use typography children without having to create their own style property for them

Reply:
agree with that,but  i think it also could be and  it was done before me,so no need to change it

Reply:
> agree with that,but i think it also could be and it was done before me,so no need to change it

I wrote how to improve it. 
It's unnecessary give own style property if we can use nested styles.
There is no need to create our own style property if we can use nested styles. You can see how we use this in our project

Reply:
In some cases creating our styles is clearer and safer because nested styles can also be acceptable for other elements if we add them in future, I don`t see a reason to change that

Reply:
> In some cases creating our styles is clearer and safer because nested styles can also be acceptable for other elements if we add them in future, I don`t see a reason to change that

But in our case, we only have a button icon and text. You can use nested styles for this and no longer have to create your own styles for icons and text.
This is good practice. We used it in our project.
I'm not sure if new items will be added here